lib/build.js

```diff
@@ -42,6 +42,11 @@ function build(clusterWorkerId) {
     }
     str = str + encode(alphabet.lookup, seconds);
 
+    // rebuild if str doesn't end in alpha-numeric character
+    while ( !/[a-zA-Z0-9]$/.test(str) ) {
```

Comment:
I don't think you need to change this module. Lots of people depend on this module and I don't think most of them need this feature. It might be best to just change the id by adding characters after it is generated in your own code, not in the code here.